src/plugins/data/public/shard_failure_modal/shard_failure_modal.test.tsx

```diff
@@ -1,28 +0,0 @@
-/*
- * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
- * or more contributor license agreements. Licensed under the Elastic License
- * 2.0 and the Server Side Public License, v 1; you may not use this file except
- * in compliance with, at your election, the Elastic License 2.0 or the Server
- * Side Public License, v 1.
- */
-
-import React from 'react';
-import { shallowWithIntl } from '@kbn/test-jest-helpers';
-import { ShardFailureModal } from './shard_failure_modal';
-import { shardFailureRequest } from './__mocks__/shard_failure_request';
-import { shardFailureResponse } from './__mocks__/shard_failure_response';
-
-describe('ShardFailureModal', () => {
```

Comment:
Would it make sense to have a similar test for IncompleteResultsModal?

Reply:
Add IncompleteResultsModal jest tests in https://github.com/elastic/kibana/pull/165512/commits/b724b9265230727c3f22568a9900d4b74caa4152